src/Files.App/Actions/Display/LayoutAction.cs

```diff
@@ -220,22 +186,57 @@ public HotKey HotKey
 		public HotKey MediaHotKey
 			=> new(Keys.OemMinus, KeyModifiers.Ctrl, false);
 
+		public bool IsExecutable
+			=> contentPageContext.PageType is not ContentPageTypes.Home;
```

Comment:
`IsExecutable` should be false when `ExecuteAsync` does nothing.

Reply:
We can add that but it's a lot of additional logic with little benifit

Reply:
It prevents meaningless commands from appearing in the command palette.